CONTRIBUTING.rst

```diff
@@ -76,8 +76,12 @@ All API specifications require a changelog entry. Adding to the changelog can on
 be done after you've opened your pull request, so be sure to do that first.
 
 The changelog is managed by `Towncrier <https://github.com/twisted/towncrier>`_ in the
-form of "news fragments". The news fragments for the client-server API are stored
-under ``changelogs/client_server/newsfragments``.
+form of "news fragments". Depending on which API you changed, an entry should be added to
+each relevant API's ``newsfragments`` directory. A directory exists for each API under
+``changelogs/``. For instance, news fragments for the client-server API are stored
+under ``changelogs/client_server/newsfragments``. Any changes to the repository that do
+not affect the spec content itself, such as changes to the build script, formatting, CSS,
+etc. should be documented under ``changelogs/internal/newsfragments``.
 
 To create a changelog entry, create a file named in the format ``prNumber.type`` in
 the ``newsfragments`` directory. The ``type`` can be one of the following:
```

Comment:
Do we want to provide any additional guidance on what types are generally used for internal changes? So far I only see "clarification" and "feature".

Reply:
Hopefully it should be relatively self-explanatory, although I'm surprised there aren't more bugfixes.